tests/loaders/test_loaders.py

```diff
@@ -0,0 +1,125 @@
+# Copyright (c) 2016-2022 Memgraph Ltd. [https://memgraph.com]
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+from unittest.mock import patch, Mock
+
+from gqlalchemy.loaders import (
+    DataLoader,
+    FileSystemHandler,
+    FileSystemTypeEnum,
+    NameMapper,
+    PyarrowDataLoader,
+    LocalFileSystemImporter,
+    get_data_loader,
+)
+
+
+class myFileSystemHandler(FileSystemHandler):
+    def __init__(self) -> None:
+        super().__init__()
+
+    def get_path(self):
+        pass
+
+
+class myDataLoader(DataLoader):
+    def __init__(self, file_system_handler: FileSystemHandler) -> None:
+        super().__init__(file_system_handler)
+        self.num = 5
+
+    def load_data(self, collection_name: str, is_cross_table: bool = False) -> None:
+        self.num = 42
+
+
+@pytest.fixture
+def dummy_loader():
+    return myDataLoader(myFileSystemHandler())
+
+
+def test_name_mapper_get_label():
+    """Test get_label from NameMapper class"""
+    mappings = {"individuals": {"label": "INDIVIDUAL"}, "address": {"label": "ADDRESS"}}
+    name_mapper = NameMapper(mappings)
+    label = name_mapper.get_label("individuals")
+
+    assert label == "INDIVIDUAL"
+
+
+def test_name_mapper_get_property_name():
+    """Test get_property_name from NameMapper class"""
+    mappings = {"individuals": {"label": "INDIVIDUAL"}, "address": {"label": "ADDRESS"}}
+    name_mapper = NameMapper(mappings)
+    property_name = name_mapper.get_property_name("individuals", "label")
+    assert property_name == "label"
+
+
+def test_file_system_handler_abstract():
+    """Test that base FileSystemHandler class can't be instantiated"""
+    with pytest.raises(TypeError):
+        FileSystemHandler()
+
+
+def test_data_loader_abstract():
```

Comment:
Remove this test

Reply:
No need for testing the language. Test simply tries to create instance of abstract class.